wurst/systems/omnitower/OmnitowerPermittedItems.wurst

```diff
@@ -1,13 +1,10 @@
 package OmnitowerPermittedItems
 import ItemExtensions
+import RegisterEvents
 
 init
-    CreateTrigger()
-    ..registerAnyUnitEvent(EVENT_PLAYER_UNIT_PICKUP_ITEM)
-    ..addCondition(Condition(() -> begin
+    registerPlayerUnitEvent(EVENT_PLAYER_UNIT_PICKUP_ITEM) () ->
         let theItem = GetManipulatedItem()
         let theUnit = GetManipulatingUnit()
         if  theUnit.getTypeId() == UNIT_OMNITOWER and theItem.getType() != ITEM_TYPE_CHARGED
             theUnit.removeItem(theItem)

```

Comment:
Does theUnit.removeItem drop the item on the ground or remove it from the game and destroy it?

Reply:
Yea, it drops the item on the ground